salt/uwsgi/files/django.ini

```diff
@@ -1,10 +1,13 @@
 [uwsgi]
 ; https://www.techatbloomberg.com/blog/configuring-uwsgi-production-deployment/
-; strict = true errors due to deb-confname in /usr/share/uwsgi/conf/default.ini
```

Comment:
I have enabled `strict` on the new hosting and I am not seeing any errors, the app loads correctly.
We probably need to prevent this being accidentally deployed to any of the older hosting - assume it still errors there.